src/theme/components/button.js

```diff
@@ -5,31 +5,37 @@ export const Button = {
       borderColor: props.colorMode === 'dark' ? 'teal.900' : 'orange.50',
       color: props.colorMode === 'dark' ? 'teal.900' : 'orange.100',
       bg: props.colorMode === 'dark' ? 'orange.100' : 'teal.800',
-      _hover: {
-        color: props.colorMode === 'dark' ? 'teal.900' : 'orange.100',
-        bg: props.colorMode === 'dark' ? 'orange.200' : 'teal.600'
-      }
+      _hover: props.disabled
+        ? null
+        : {
+            color: props.colorMode === 'dark' ? 'teal.900' : 'orange.100',
+            bg: props.colorMode === 'dark' ? 'orange.200' : 'teal.600'
+          }
     }),
     secondary: (props) => ({
       borderWidth: 2,
       borderColor: props.colorMode === 'dark' ? 'orange.100' : 'teal.900',
       color: props.colorMode === 'dark' ? 'orange.100' : 'teal.900',
       bg: props.colorMode === 'dark' ? 'teal.800' : 'orange.50',
-      _hover: {
-        bg: props.colorMode === 'dark' ? 'teal.600' : 'orange.100'
-      }
+      _hover: props.disabled
+        ? null
+        : {
+            bg: props.colorMode === 'dark' ? 'teal.600' : 'orange.100'
+          }
     }),
```

Comment:
```suggestion
      _hover: {
            bg: props.colorMode === 'dark' ? 'teal.600' : 'orange.100',
            _disabled: null
          }
    }),
```

Reply:
https://user-images.githubusercontent.com/39521158/181189873-b6a76ad5-468a-4fe3-87e5-536e32cbf97e.mov

it change hover but didn't remove it finally

Reply:
https://user-images.githubusercontent.com/39521158/181221252-2f17dfac-4560-45b9-a26a-f10683100198.mov

(_disabled: null) didn't work properly, so I have to add styles to _disabled instead of null